app/views/additional_details/submissions/_submission_info.html.haml

```diff
@@ -9,6 +9,10 @@
             %p.list-group-item-text
               - if item.item_type == 'multiple_dropdown'
                 = qr.content.delete('[]').delete('\"')
```

Comment:
It looks at though it's performing the same function as the checkbox gsub. @wtholt is that correct? If so, could we combine the checkbox and multiple_dropdown logic into one if statement
`if ['multiple_dropdown', 'checkbox'].include?(item.item_type)`